src/NuGet.Core/NuGet.Versioning/FloatRange.cs

```diff
@@ -164,7 +164,7 @@ public bool Satisfies(NuGetVersion version)
         /// </summary>
         public static FloatRange Parse(string versionString)
         {
-            TryParse(versionString, out FloatRange range);
+            _ = TryParse(versionString, out FloatRange range);
```

Comment:
I think generally `Parse` methods throw exception if parse fails.

Reply:
Yes, that is true for just `Parse`. But here it's `TryParse`, for example `int32.TryParse` doesn't throw.
Here `range = null` implicitly signaling parsing didn't succeed.